config/metallb/metal-lb.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: metallb.io/v1beta1
```

Comment:
Maybe put it in `./examples`?

Reply:
sure done

Reply:
I see you've opted to remove the file altogether.